### DISCUSSION TITLE
Broken search snippets with Sphinx 7.2.2 ?

### DISCUSSION THREAD

**Discussion by lalmeras** [pradyunsg/furo#706] [Potential Issue] [answered]
I step into this issue with Sphinx 7.2.2 ; search snippets are missing in search results, with 404 errors triggered by searchtools.js. It seems that html GET used to build search snippets are broken (undefined<WORKING_URL>).

When I check searchtools.js and Sphinx documentation (where search snippets are working), I notice that there is a `data-content_root` that is used by searchtools.js (`document.documentElement.dataset.content_root`).

I think this is an added item in sphinx, in their own template, that may be added to `base.html` furo layout.

If I manually modify `base.html` to add `data-content_root=""` it fixes the issue.

Are there anyone else encountering this issue ? Did I miss something on my side ?

**Comment by pradyunsg** [comment]
Got a reproducer for this?

**Reply by pradyunsg** [reply]
Oh, fun. Only seems like this only happens with `dirhtml` builds.

**Reply by pradyunsg** [reply]
https://github.com/sphinx-doc/sphinx/issues/11608#issuecomment-1684410476

**Comment by lalmeras** [comment, marked as answer]
Hi,

I just test the 2023.8.19 release with Sphinx 7.2.2 and it fixes the issue.

Thank you for your quick feedback and this very fast fix.
